master/internal/resourcemanagers/kubernetes/events.go

```diff
@@ -63,7 +65,8 @@ func (e *eventListener) Receive(ctx *actor.Context) error {
 }
 
 func (e *eventListener) startEventListener(ctx *actor.Context) {
-	events, err := e.clientSet.CoreV1().Events(e.namespace).List(metaV1.ListOptions{})
+	events, err := e.clientSet.CoreV1().Events(e.namespace).List(
+		context.TODO(), metaV1.ListOptions{})
```

Comment:
from the doc string of `context.TODO()`, which I agree with
```
// TODO returns a non-nil, empty Context. Code should use context.TODO when
// it's unclear which Context to use or it is not yet available (because the
// surrounding function has not yet been extended to accept a Context
// parameter).
```

Reply:
(note, the ctx inside `*actor.Context` is meaningless, just used for dumb tracing stuff i added ages ago)